util/src/main/java/io/kubernetes/client/util/ClientBuilder.java

```diff
@@ -117,6 +121,24 @@ private static File findConfigFromEnv() {
     }
   }
 
+  private static class KubeConfigEnvParser {
+    private String parseKubeConfigPath(String kubeConfigEnv) {
+      if (kubeConfigEnv == null) {
+        return null;
+      }
+
+      final String[] filePaths = kubeConfigEnv.split(File.pathSeparator);
+      final String kubeConfigPath = filePaths[0];
+      if (filePaths.length > 1) {
+        log.info(
```

Comment:
I think this should be log.warning, not log.info since we're explicitly doing something that isn't quite what the user expects...

Reply:
Yeah, you are right, fixed.